lib/pacemaker/pcmk_sched_colocation.c

```diff
@@ -1249,13 +1247,12 @@ is_nonempty_group(pe_resource_t *rsc)
  * \param[in]     factor      Incorporate scores multiplied by this factor
  * \param[in]     flags       Bitmask of enum pe_weights values
  *
- * \return Nodes, with scores modified by this constraint
  * \note This function assumes ownership of the nodes argument. The caller
  *       should free the returned copy rather than the original.
```

Comment:
No longer a returned copy but this is addressed in a later renaming commit

Reply:
Whoops, I'll change that in the same commit